OptimizelySDKCore/OptimizelySDKCore/OPTLYEventMetric.h

```diff
@@ -18,9 +18,9 @@
 
 #import <Foundation/Foundation.h>
 #ifdef UNIVERSAL
-    #import "JSONModelLib.h"
+    #import "OPTLYJSONModelLib.h"
```

Comment:
can this all be moved to a prefix header?

Reply:
that way the next time we need a refactor like this, we will only need to change the prefix header

Reply:
Possible CON's to the prefix header proposal:

* Apple and Apple fans are discouraging *.pch's these days.
"Why precompiled header dropped in recent Xcode versions?
... Correct answer by Justin [410 points - Apple Staff]
Precompiled Header Files are discouraged in favor of modules. ..."
https://forums.developer.apple.com/thread/22060
"4 Ways Precompiled Headers Cripple Your Code"
https://qualitycoding.org/precompiled-header/

* Typically, the single *.pch can't be made 'Public', but we are creating a *.framework
being distributed to the public.  Multiple *.h can be sorted into 'Public' and 'Private'.

* Knowing this P.R. would already involve a lot of change and commits, I've been
seeking to not bite off more than what's required to achieve the PULL REQUEST PURPOSE.
Creating *.pch is unnecessary for the PURPOSE but the rename is already necessary and
suffices for this line of code.